src/megatron/bridge/models/gpt_oss/gpt_oss_provider.py

```diff
@@ -17,14 +17,20 @@
 from typing import Callable, List, Literal, Optional, Tuple, Union
 
 import torch
-from megatron.core.fusions.fused_bias_geglu import quick_gelu
 from megatron.core.models.gpt import GPTModel as MCoreGPTModel
 from megatron.core.transformer.enums import AttnBackend
 from megatron.core.utils import is_te_min_version
 
 from megatron.bridge.models.gpt_provider import GPTModelProvider
 
 
+try:
+    from megatron.core.fusions.fused_bias_geglu import quick_gelu
+except ImportError:
+    # Fallback if fused_bias_geglu is not available
+    quick_gelu = torch.nn.functional.gelu
```

Comment:
the import fallbacks are hard to follow since they don't line up with the MLM submodule commit pinned in the repo now. i understand its done to manage RL's own MLM dependency but could you leave a comment/TODO to remove this once RL's separate MLM pinned commit is resolved?

Reply:
cool